pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/PersistentTopicsBase.java

```diff
@@ -1330,10 +1322,15 @@ protected void internalGetManagedLedgerInfo(AsyncResponse asyncResponse, boolean
                                     asyncResponse.resume(new RestException(t));
                                 }
                             }
+                            PartitionedManagedLedgerInfo partitionedManagedLedgerInfo =
```

Comment:
Do we need to add `else` block for line 1325 ~1333 ?

Reply:
Adding `else` is more rigorous. LGTM

Reply:
thanks. fixed. @Technoboy- PTAL